src/utils/mock_data/createJoinRequests.js

```diff
@@ -0,0 +1,42 @@
+const { pool } = require('../db');
+const joinRequests = require('./joinRequests');
+
+const createJoinRequests = async () => {
+    try {
+
+        const { rowCount } = await pool.query(`SELECT id FROM "ProjectJoinRequest" LIMIT 20`);
+        if (rowCount === 20) {
+            return;
+        }
+
+        let promises = [];
+
+        for (const joinRequest of joinRequests) {
+            promises.push(
+                pool.query(
+                    `INSERT INTO "ProjectJoinRequest"(project_id, username, message) VALUES($1, $2, $3)`,
```

Comment:
You insert into the `project_id` column the joinRequest id, is this a typo?

Reply:
I should have made it more clear, but `joinRequest.id` is the project ID. I didn't manually specify the request ID because I didn't need to reference a specific request anywhere except in one spot (the one request denial) which I just referenced using the project id and username since that pairing has to be unique.